episodes/compare-interventions.Rmd

```diff
@@ -274,21 +588,137 @@ We recommend to read the vignette on [Modelling responses to a stochastic Ebola
 
 :::::::::::::::::::::::::::
 
-<!-- ## Comparing scenarios -->
 
-<!-- *Coming soon* -->
 
 
 
-<!-- ## Challenge : Ebola outbreak analysis  -->
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Challenge : Ebola outbreak analysis
+
+You have been tasked to investigate the potential impact of an intervention on an Ebola outbreak in Guinea. Using `model_ebola()` and the the information detailed below, find the number of infections averted when :
+
++ an intervention is applied to reduce transmission rate by 0.5 from day 60 and,
```

Comment:
```suggestion
+ an intervention is applied to reduce the transmission rate by 50% from day 60 and,
```

Reply:
Have swapped to %, as otherwise might be misinterpreted as absolute change of 0.5?